### PR TITLE
Version 0.1.5 bump, minor changes to make this work

### LINKED ISSUE #2: The plugin does not work as it has missing dependencies
As noted from https://github.com/elastic/logstash/issues/3258, this plugin has missing dependencies so it does not really work....

### PR DESCRIPTION
**Pull request by purbon**
Fixes #2 and make this plugin work (it had missing dependencies). It also include a small registry test.

### CODE REVIEW

File: spec/outputs/websocket_spec.rb [hunk 1]
```diff
@@ -1 +1,11 @@
+# encoding: utf-8
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/plugin"
+
+describe "output/websocket" do
+
+  it "should register" do
+    input = LogStash::Plugin.lookup("output", "websocket").new()
```
Comment (jsvd) [inline spec/outputs/websocket_spec.rb]: why not just `subject(:input) { LogStash::Outputs::Websocket.new }`?

Comment (purbon) [inline spec/outputs/websocket_spec.rb]: no special reason, I will remove the parentheses :+1:

Comment (jsvd) [inline spec/outputs/websocket_spec.rb]: also..it's an output, not an input :)

Comment (purbon) [inline spec/outputs/websocket_spec.rb]: LOL! good catch, c&p is very bad.

**Comment by purbon** [comment]
@jsvd thanks for your review man! got all your changes proposals in.

File: logstash-output-websocket.gemspec [hunk 2]
```diff
@@ -21,7 +21,8 @@ Gem::Specification.new do |s|
 
   # Gem dependencies
   s.add_runtime_dependency "logstash-core", '>= 1.4.0', '< 2.0.0'
-
+  s.add_runtime_dependency 'logstash-codec-plain'
+  s.add_runtime_dependency 'sinatra'
```
Comment (jsvd) [inline logstash-output-websocket.gemspec]: add a dependency restriction in sinatra please

Comment (purbon) [inline logstash-output-websocket.gemspec]: kk, will do that.

**Comment by jsvd** [comment]
LGTM

**Pull request closed without merging**